README.md

```diff
@@ -1,94 +1,14 @@
-# Friday Night Funkin
+# Friday Night Funkin Discord RPC

```

Comment:
You should revert README back to what it was beforehand.

Reply:
Ah, I forgot about that. Thanks.